README.md

```diff
@@ -240,6 +241,58 @@ Book.search('*', sort: ['title:asc'])
 
 👉 Don't forget to set up the `sortable_attributes` option in the `meilisearch` block of your model.
 
+## 🔍🔍 Multi search
+
+MeiliSearch supports searching multiple models at the same time (see [🔍 Custom search](#-custom-search) for search options):
```

Comment:
```suggestion
Meilisearch supports searching multiple models at the same time (see [🔍 Custom search](#-custom-search) for search options):
```

Reply:
Ooops sorry, old habits die hard lol

Reply:
hahaha no issue!

Also, if you have time you could put in your big breaking changing the main class name to `Meilisearch` instead of `MeiliSearch`. Not only here but in the ruby gem as well.

Reply:
That should be easy, just a find and replace. But it will be a huge breaking change, possibly v1?

Reply:
I don't want to think about making a v1 right now, especially since we will introduce a lot of new stuff. Let's do all the breaking and wait for them to sink into the users. Also we will be able to understand if the changes had the effect we wanted :)